desktop_version/src/LevelDebugger.cpp

```diff
@@ -0,0 +1,428 @@
+                render_info(line++, "Draw Frame", help.String(entity->drawframe));
+                render_info(line++, "Size", help.String(entity->size));
+                render_info(line++, "Direction", help.String(entity->dir));
+
+                line++;
+
+                // Mostly contains duplicates, but for ease of use
+                switch (entity->type)
+                {
+                case 0:
+                    // Player
+                    render_info(line++, "Gravity", help.String(game.gravitycontrol));
+                    render_info(line++, "Checkpoint", help.String(game.savepoint));
+                    break;
+                case 1:
+                    // Moving platforms and enemies
+                    render_info(line++, "Speed", help.String(entity->para));
+                    render_info(line++, "Movement type", help.String(entity->behave));
+                    break;
+                case 7:
+                    // Trinkets
+                    render_info(line++, "ID", help.String(entity->para));
+                    break;
+                case 8:
+                    // Checkpoints
+                    render_info(line++, "ID", help.String(entity->para));
+                    render_info(line++, "Active", game.savepoint == entity->para ? "True" : "False");
+                    break;
+                case 9:
+                    // Horizontal gravity lines
+                    render_info(line++, "Horizontal");
+                    break;
+                case 10:
+                    // Vertical gravity lines
+                    render_info(line++, "Vertical");
+                    break;
+                }
+
+
+                graphics.draw_rect(hover_box.x, hover_box.y, hover_box.w, hover_box.h, graphics.getRGB(32, 255 - help.glow, 255 - help.glow));
+            }
+            else
+            {
+                blockclass* block = &obj.blocks[hovered];
+                render_info(line++, "Index", help.String(hovered));
+                render_coords(line++, "Position", block->rect.x, block->rect.y);
+                render_coords(line++, "Size", block->rect.w, block->rect.h);
+
+                line++;
+
+                if (block->type == TRIGGER || block->type == ACTIVITY)
+                {
+                    render_info(line++, "Script", block->script);
+                    render_info(line++, "State", help.String(block->trigger));
+                }
+                else if (block->type == DIRECTIONAL)
+                {
+                    render_info(line++, "Direction", help.String(block->trigger));
+                }
+
+                graphics.draw_rect(hover_box.x, hover_box.y, hover_box.w, hover_box.h, graphics.getRGB(255 - help.glow, 32, 32));
+            }
+        }
+
+        font::print(PR_BOR | PR_FONT_8X8, 5, 14, loc::gettext("[Press TAB to toggle movement]"), 220 - (help.glow), 220 - (help.glow), 255 - (help.glow / 2));
```

Comment:
If a string is translatable, it shouldn't be `PR_FONT_8X8`.
```suggestion
        font::print(PR_BOR, 5, 14, loc::gettext("[Press TAB to toggle movement]"), 220 - (help.glow), 220 - (help.glow), 255 - (help.glow / 2));
```